Dockerfile

```diff
@@ -205,6 +205,8 @@ RUN mkdir -p /dart-protobuf && \
     cd /dart-protobuf/protoc_plugin && pub install && dart2native --verbose bin/protoc_plugin.dart -o protoc_plugin && \
     install -D /dart-protobuf/protoc_plugin/protoc_plugin /out/usr/bin/protoc-gen-dart
 
+FROM moul/protoc-gen-gotemplate:v${GOTEMPLATE_VERSION} as protoc_gotemplate
```

Comment:
Please ensure there are 2 newlines above the `FROM` statement to delimit these sections